build/package/chocolatey/keboola-cli.nuspec

```diff
@@ -3,10 +3,11 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>keboola-cli</id>
-    <version>2.2.0</version>
+    <version>{VERSION}</version>
     <packageSourceUrl>https://github.com/keboola/keboola-as-code/tree/main/build/package/chocolatey</packageSourceUrl>
     <title>Keboola CLI</title>
     <authors>Keboola</authors>
+    <iconUrl>https://assets.website-files.com/5e21dc6f4c5acf29c35bb32c/5e7263a8bf518b1d3be1760a_256.png</iconUrl>
```

Comment:
Chtějí tam ikonku alespoň 128px a musí být v cdnku, tak jsem zatím použil favikonu z keboola.com (https://assets.website-files.com/5e21dc6f4c5acf29c35bb32c/5e7263a8bf518b1d3be1760a_256.png). Měla by být asi na tom profilu vlevo (https://community.chocolatey.org/packages/keboola-cli) ale zatím je tam nějaký jejich placeholder, asi to mají nacachovaný.

Pokud ti přijde ok, tak ji potom zkopíruju do našeho kbc-ui assets, nevím jak moc persistentní tady to úložiště je.